girder/api/v1/assetstore.py

```diff
@@ -150,6 +150,10 @@ def createAssetstore(self, params):
                enum=('folder', 'collection', 'user'))
         .param('progress', 'Whether to record progress on the import.',
                dataType='boolean', default=False, required=False)
+        .param('fileIncludeRegex', 'If set, only filenames matching this regular '
+               'expression will be imported.', required=False)
+        .param('fileExcludeRegex', 'If set, only filenames that do not match this regular '
+               'expression will be imported.', required=False)
```

Comment:
May be good to clarify with "If a filename matches both inclusion and exclusion patterns, exclusion takes precedence over inclusion." in the docs here.